README.md

```diff
@@ -137,7 +137,6 @@ above, please let us know! We'd love to add your project to the list!
 
 [`open-telemetry/opentelemetry-rust`]: https://github.com/open-telemetry/opentelemetry-rust
 [`opentelemetry-jaeger`]: https://crates.io/crates/opentelemetry-jaeger
-[`Jaeger`]: https://www.jaegertracing.io
```

Comment:
Not sure why you removed an associated project. They are also implementing OTLP.

Reply:
This is not the list of associated project. Its just list of links used in the doc, and jaeger was duplicated (and so removed)